control/controlclient/map.go

```diff
@@ -666,6 +666,8 @@ func peerChangeDiff(was tailcfg.NodeView, n *tailcfg.Node) (_ *tailcfg.PeerChang
 		case "CapMap":
 			if n.CapMap != nil {
 				pc().CapMap = n.CapMap
```

Comment:
But now that I see this code, this is wrong. If these are equal, it shouldn't be a diff.